create-container.c

```diff
@@ -351,6 +353,7 @@ static struct option const opts[] = {
 	{ "dumpContrHdr",     required_argument, 0,  '0' },
 	{ "dumpPrefixHdr",    required_argument, 0,  '1' },
 	{ "dumpSwHdr",        required_argument, 0,  '2' },
+	{ "security-version", required_argument, 0,  'S' },
```

Comment:
the "required_argument" just means "if someone uses -S , it must follow with a version" right?   Vs. .. the "-S" must always be supplied when command is invoked?

Reply:
@bofferdn That is my understanding - "required_argument" means that a value must be provided with the flag.

I tested this code without invoking the --security-version flag and it produced the expected behavior - the version was set to 0.